.storybook/main.js

```diff
@@ -1,17 +1,123 @@
 const path = require('path');
+const argv = require('yargs').argv;
+
+const IS_PRODUCTION = Boolean(argv.production);
+const VERSION = argv.production ? pkg.version : 'dev';
+const SOURCE_DIR = path.join(__dirname, '../src');
+const SOURCE_DOCS_DIR = path.join(SOURCE_DIR, '_docs');
+const SOURCE_COMPONENTS_DIR = path.join(SOURCE_DIR, 'components');
 
 module.exports = {
-  stories: ['../src/*.stories.@(jsx|mdx)', '../src/**/*.stories.@(jsx|mdx)'],
-  addons: ['@storybook/addon-docs', '@storybook/addon-essentials', '@storybook/addon-actions', '@storybook/addon-links'],
+  stories: ['../src/**/*.stories.@(jsx|mdx)'],
+  addons: [
+    '@storybook/addon-a11y',
+    '@storybook/addon-essentials',
+    '@storybook/addon-links',
+  ],
   webpackFinal: (config) => {
-
     // change 'sideEffects' flag to true in package.json in order to include scss files in static build
-    config.module.rules.push({
-      test: /\.scss$/,
-      use: ['style-loader', 'css-loader', 'sass-loader'],
-      include: path.resolve(__dirname, '../src/main.scss'),
-    });
 
-    return config;
-  }
+    const _config = {
+      ...config,
+      module: {
+        ...config.module,
+        rules: [
+          ...config.module.rules.slice(1),
```

Comment:
could you add comment, what you're removing?

Reply:
yea, good point